ui/app/adapters/sync/association.js

```diff
@@ -24,6 +22,26 @@ export default class SyncAssociationAdapter extends ApplicationAdapter {
     return this.ajax(url, 'GET');
   }
 
+  // array of association data for each destination a secret is synced to
+  fetchSyncStatus({ mount, secretName }) {
+    const url = `${super.buildURL()}/associations/${mount}/${secretName}`;
+    return this.ajax(url, 'GET').then((resp) => {
+      const { associated_destinations } = resp.data;
+      const syncData = [];
+      for (const key in associated_destinations) {
+        const data = associated_destinations[key];
+        // renaming keys to match query() response
```

Comment:
This isn't necessary because this data doesn't map to an ember data model, but it does make the template data easier to discern, plus creates parity with the association model by using the same naming